src/settings.cpp

```diff
@@ -2,10 +2,10 @@
 
 using namespace cubos;
 
-Settings* Settings::instance = nullptr;
-
 Settings* Settings::getInstance()
 {
+    static Settings* instance = nullptr;

```

Comment:
Why a pointer? This will lead to a memory leak anyway.
Since we're doing it this way we can declare it the normal way and just return a reference to it, like
```cpp
Settings& Settings::getInstance()
{
    static Settings instance;
    return instance;
}